Sources/GRPCProtobuf/ContiguousBytesAdapter.swift

```diff
@@ -63,3 +63,6 @@ struct ContiguousBytesAdapter<
     try self.bytes.withUnsafeMutableBytes(body)
   }
 }
+
+@available(*, unavailable)
+extension ContiguousBytesAdapter: Sendable {}
```

Comment:
This isn't quite right: `ContiguousBytesAdapter` is `Sendable` when `Bytes` is `Sendable`